cmd/kube-apiserver/app/options/options.go

```diff
@@ -134,14 +134,32 @@ func NewServerRunOptions() *ServerRunOptions {
 	return &s
 }
 
+func addDummyInsecureFlags(fs *pflag.FlagSet) {
```

Comment:
I'd probably set v1.24 as a target removal version in the message and in a TODO

Reply:
added